src/createStore.ts

```diff
@@ -39,35 +39,54 @@ import { kindOf } from './utils/kindOf'
  * @returns A Redux store that lets you read the state, dispatch actions
  * and subscribe to changes.
  */
+
+// 3 overloaded functions
+// 1st overload
+//     v--'defualt' allows createStore function to be called by different name when imported later 
```

Comment:
Couple of questions:

- What does `v--` mean here?
- Are you talking about the `default` keyword?

Reply:
It was my attempt at making a down arrow, but I think it did not work LOL